metadata-io/src/test/java/com/linkedin/metadata/search/LineageSearchServiceTest.java

```diff
@@ -87,6 +88,8 @@ public class LineageSearchServiceTest extends AbstractTestNGSpringContextTests {
   private ESIndexBuilder _esIndexBuilder;
   @Autowired
   private SearchConfiguration _searchConfiguration;
+  @Autowired
+  private CustomSearchConfiguration _customSearchConfiguration;
```

Comment:
Just curious - why are there 2 things now instead of extending searchConfiguration?

Reply:
so `SearchRequestHandler` requires both `_searchConfiguration` and `_customSearchConfiguration` - honestly not positive about the need for two things but i assume they're just configs coming from different sources. ESBrowseDao needs `SearchRequestHandler` in order to generate filters properly with our `orFilters` - something old browse doesn't support.

this test file needs it because `buildEntitySearchService` constructs `ESBrowseDAO`